account_mtd/models/mtd_api_request_tracker.py

```diff
@@ -0,0 +1,13 @@
+from odoo import models, fields, api
+
+class MtdApiRequestTracker(models.Model):
+    _name = 'mtd.api_request_tracker'
+    _description = "authorisation request table"
+    
+    user_id = fields.Integer('User Id', required=True)
```

Comment:
camel-cased string parameter for field is implied. i.e if you have a field `my_field`, the string will default to `My Field`. Hence the string `User Id` here is unnecessary and can be removed